lib/messageEvents.js

```diff
@@ -1,33 +1,22 @@
 'use strict';
 
-var api = 'message-events';
+const api = 'message-events';
```

Comment:
That is correct. We are dropping support for 0.10 and 0.12. If they are still using those versions, they can use version 1.x of the library.